README.md

```diff
@@ -65,6 +65,39 @@ will assist in catching regressions if the `update` function begins to set other
 values or if the view code does not capture newly-added fields to the `User`
 schema.
 
+For the sake of example, if the controller action under test returned a `User`
+with the same keys and values differing only with the value for id, then the
+test would fail and the output may look like the example below:
+
+```
+  1) test success: it returns a 200 and a newly updated User (UserControllerTest)
+     test/user_controller_test.exs:294
+     There were issues the comparison:
+
+     Values did not match for:
+       field: :id
+         expected: "1ee3e9c4-fa81-4612-a13a-c554e5a3d438"
+         actual: "a2ab48fa-96fb-41e5-8f2d-2f94f47fef91"
+     code: assert_values_for(expected: expected_updates, actual: {returned_user :string_keys}, fields: fields_for(User))
```

Comment:
We replicated the output because we did not have the phoenix application where the demo test came from.

This faithfully copies the example code it refers to, but we do not know what lines ExUnit will truly return.